packages/frontend/src/pages/home/hero/index.tsx

```diff
@@ -28,25 +30,24 @@ export const Hero = () => {
     const { t } = useTranslation();
     const { loading, kpiTokens } = useFeaturedKPITokens();
     const videoRef = useRef<HTMLVideoElement>(null);
+    const modalIsOpen = useSelector<HostState>((state) => state.modals.isOpen);
 
     useEffect(() => {
         if (!videoRef || !videoRef.current) return;
         const currentRef = videoRef.current;
-        return () => {
-            currentRef.pause();
-        };
-    });
+        modalIsOpen ? currentRef.pause() : currentRef.play();
```

Comment:
I'd avoid playing the video when the modal closes, otherwise every time the user opens and closes a model the video starts even if it was previously over.
I'd just stop it if any modal gets opened but then leave the choice to the user to start the video manually.

Reply:
Yes, that might be better.

Reply:
fixed 2059e7e1c1e6215fab40c71b46b81713a95385a8